features/verblijfplaatshistorie.feature

```diff
@@ -0,0 +1,169 @@
+# language: nl
+
+Functionaliteit: Tonen van verblijfplaatsen
+
+  Huidige en vorige verblijfplaatsen van ingeschreven personen kunnen worden opgezocht.
+
+  De gebruiker kan de verblijfplaats raadplegen op een specifieke peildatum (in het verleden).
+  De gebruiker kan de verblijfplaatsen raadplegen over een specifieke periode.
+  Filteren op peildatum of periode wordt gedaan op basis van de datum aanvang adreshouding van de verblijfplaats(en).
+  Op periode gefilterde gegevens tonen alle verblijfplaatsen waar de persoon gedurende de periode heeft verbleven. Ook wanneer de persoon al voor de periode op een verblijfplaats verbleef en binnen de gevraagde periode nog steeds, wordt de verblijfplaats opgenomen in het antwoord. Ook wanneer de persoon tijdens (een deel van) de periode op een verblijfplaats verbleef en na de gevraagde periode nog steeds, wordt de verblijfplaats opgenomen in het antwoord.
+
+  Wanneer de datum aanvang gedeeltelijk onbekend is, wordt voor de filtering aangenomen dat de persoon gedurende de gehele onzekerheidstijd op het adres heeft verbleven. Wanneer van de datum aanvang adreshouding alleen het jaar bekend is, wordt aangenomen dat de persoon het hele jaar op deze verblijfplaats heeft verbleven.
+  Wanneer van de datum aanvang adreshouding alleen het jaar en de maand bekend is, wordt aangenomen dat de persoon de hele maand op deze verblijfplaats heeft verbleven.
+  Wanneer de datum aanvang adreshouding geheel onbekend is, wordt voor de filtering aangenomen dat de persoon daar altijd al woonde.
+
+  Bij een historische verblijfplaats wordt een datumTot opgenomen, die gelijk is aan de datum aanvang adreshouding van de daarop volgende (actuele of historische) verblijfplaats.
+
+  Wanneer in de registratie voor de persoon meer dan één verblijfplaats, actueel (categorie 8) of historisch (categorie 58), bevat met exact dezelfde datum aanvang adreshouding, dan wordt alleen het voorkomen met de meest recente datum ingang geldigheid opgenomen. Dit speelt bijvoorbeeld wanneer er een correctie is uitgevoerd op de verblijfplaats, of wanneer er een adres hernummering is gedaan.
+
+  Historische voorkomens die onjuist zijn worden niet opgenomen in het antwoord.
+
+  De verblijfplaatsen worden in het antwoord aflopend gesorteerd op datum aanvang adreshouding, zodat de meest actuele bovenaan staat.
+
+  Wanneer een verblijfplaats een buitenlands adres betreft, wordt Datum aanvang adres buitenland (element 13.20) gebruikt in plaats van datum aanvang adreshouding. Dit betreft de sortering in het antwoord, de filtering op peildatum of periode, en het afleiden van de datumTot.
```

Comment:
Is het misschien ook een idee om Datum aanvang adres buitenland (element 13.20) ook als resultaat te mappen naar aanvangadres van Verblijfplaats? Volgens mij is het nuttig om ook voor een buitenlands adres de datum van aanvang aan de gebruiker te tonen.

Reply:
Goed idee! Het betekent tenslotte hetzelfde.